deepcell/metrics.py

```diff
@@ -339,8 +387,8 @@ def _linear_assignment(self):
         # Identify direct matches as true positives
         correct_index = np.where(self.cm_res[:self.n_true, :self.n_pred] == 1)
         self.correct_detections += len(correct_index[0])
-        self.correct_indices['y_true'].append(correct_index[0])
-        self.correct_indices['y_pred'].append(correct_index[1])
+        self.correct_indices['y_true'] += list(correct_index[0] + 1)
+        self.correct_indices['y_pred'] += list(correct_index[1] + 1)
```

Comment:
Concatenating lists is slow. Try using `list.extend` (for adding multiple values) or `list.append` (for adding single values) instead.

Reply:
Yeah, that's what morgan had before: 
![image](https://user-images.githubusercontent.com/13770365/78732614-eefc7880-78f7-11ea-8181-2d528bb8d711.png)

However, it was resulting in an output that was an array, and inside the array a list, and inside the list the values we wanted: [list([1, 2, 4)]. Sometimes we're adding a single value and sometimes multiple.

Reply:
What are the possible value types of `correct_index[0]`? Maybe we can enforce that it is always the same type? (always cast to a list, then call `.extend()`)

We should avoid using `list += list()` as it is very slow and creates duplicate lists.

Reply:
Got it. It's always a np.array, so we should always be able to cast to list. Changing now